modules/compiler/spirv-ll/test/spvasm/CMakeLists.txt

```diff
@@ -2238,6 +2238,7 @@ function(ca_assemble_spirv_ll_lit_test)
     # Generate .spv from .spvasm source.
     file(RELATIVE_PATH spvOut ${CMAKE_BINARY_DIR} ${spv})
     add_custom_command(OUTPUT ${spv}
+      COMMAND ${CMAKE_COMMAND} -E make_directory ${CMAKE_CURRENT_BINARY_DIR}/${dir}
```

Comment:
is this okay if the directory already exists?

Reply:
https://cmake.org/cmake/help/latest/manual/cmake.1.html#cmdoption-cmake-E-arg-make_directory
``` quote
make_directory <dir>...

    Create <dir> directories. If necessary, create parent directories too. If a directory already exists it will be silently ignored.

    New in version 3.5: Support for multiple input directories.
```

Reply:
According to https://cmake.org/cmake/help/v3.10/command/make_directory.html, this is a deprecated command.

Reply:
I think that refers to the "command" in CMake. Here we're using the "command-line tool" `make_directory`, which isn't marked as such. At least not in latest the documentation I linked above.

Reply:
fair enough, cmake can be confusing.